Configuration/python/CollectionProducer_cff.py

```diff
@@ -312,6 +313,14 @@ class CollectionProducer:
     EERecHits          =  cms.InputTag  ("reducedEcalRecHitsEE"),
```

Comment:
just to make sure I understand, you want to leave the AOD rec hits as default and only use miniAOD selectively?

Reply:
Exactly. I'll try doing something more permanent/automatic, but will leave it for a future PR if we actually go with the MiniAOD-only approach (after checking the deepsets case). I added that here now to not forget this approach, which is very simple to avoid using AOD somewhere.

Reply:
Talked in private with @carriganm95 and decided that this PR can be merged.